docs/designs/ps_design.md

```diff
@@ -1,53 +1,62 @@
 # Parameter Server Design
 
-
 ## Overview
-Currently, there is one parameter server (PS) co-existed with the master. In order to support PS with multiple nodes and PS fault tolerance, we need to separate PS from the master. Besides a KV store for model parameters, PS should also support updating parameters using gradients. Currently, models have two kinds of parameter, variable and embedding vector. PS should support model variable update and embedding table sparse update.
 
-The master will create a PS with *N* PS nodes, where *N* is specified by the user. Each model variable and embedding vector has a corresponding PS node. Thus, every PS node has a subset of model variables and embedding tables.
+A typical parameter server architecture of data parallelism distributed training system contains two roles:
```

Comment:
It seems that enumeration prevents us from explaining the relationship between the bullets -- workers and parameter servers.

> Many distributed training systems use multiple works and the data parallelism architecture. Data parallel means that each worker computes the opinion about how to update the model, say, gradients, from its local data. So we need a mechanism to help workers exchange their opinions. The parameter server is such an idea for gradient aggregation and model updates.